plugin/trino-base-jdbc/src/main/java/io/trino/plugin/jdbc/JdbcTableHandle.java

```diff
@@ -35,7 +35,7 @@
 public final class JdbcTableHandle
         implements ConnectorTableHandle
 {
-    private final JdbcRelationHandle relationHandle;
+    private final JdbcRelation relationHandle;
```

Comment:
could you please doublecheck for parameters, variables and methods which could contain `Handle` through out the project?